carmin-server/server/test/test_server_config.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+import copy
+from server import properties_validation
+from server.common.exceptions import MissingRequiredParameterError
+
+CONFIG_DATA = {
```

Comment:
Could isolate a bunch of test config_data in their own module instead of directly in the test. Could be reused in the case of system tests requiring a running server for example.